CMakeLists.txt

```diff
@@ -9,6 +9,9 @@ endif()
 if(NOT CMAKE_OSX_DEPLOYMENT_TARGET)
     set(CMAKE_OSX_DEPLOYMENT_TARGET "10.13" CACHE STRING "Minimum OS X version to target for deployment" FORCE)
 endif()
+if(CMAKE_OSX_DEPLOYMENT_TARGET VERSION_LESS 10.13)
+    message(FATAL_ERROR "Targeting macOS versions before 10.13 is not supported, please adjust CMAKE_OSX_DEPLOYMENT_TARGET (${CMAKE_OSX_DEPLOYMENT_TARGET})")
```

Comment:
Given that "10.13" is now mentioned 3 times, might make sense to introduce a separate variable for it.

Reply:
I'd add it even if all 3 were on the same line.

Reply:
Done with a `function`, in order to not affect the global scope.
https://stackoverflow.com/questions/31037882/whats-the-cmake-syntax-to-set-and-use-variables